platform/terraform/main.tf

```diff
@@ -84,6 +84,45 @@ resource "azurerm_cosmosdb_sql_database" "cosmosdb-container" {
   resource_group_name = azurerm_resource_group.resource-group.name
 }
 
+resource "random_password" "sql_admin_password" {
```

Comment:
I'm not sure this meets the password policy for SQL

https://learn.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver16&redirectedfrom=MSDN

Reply:
Think we might need to set "override_special" with a given set of chars

Reply:
What characters do you suggest? MSDN doesn't provide a definitive list

Reply:
I'd suggest `!@#$*()-_=+` to avoid escape issues.